packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -216,7 +216,7 @@ export interface VideoGalleryProps {
   strings?: Partial<VideoGalleryStrings>;
   /**
    * Maximum number of participant remote video streams that is rendered.
-   * @defaultValue 4
+   * @defaultValue 9
```

Comment:
@PorterNan would this be considered a breaking change? Should we leave the default as `4` in the VideoGallery, but at the composite level choose 9?

Reply:
I think we can keep the default value at 4. The actual OVC value will be larger and that is fine.. up to 9.. eventually up to 25, and 49 but if we have 4 and it snaps to this if OVC fails then we should be ok?